Source/Classes/AnimatedImages/PINAnimatedImageView.m

```diff
@@ -61,6 +61,31 @@ - (void)commonInit:(PINCachedAnimatedImage *)animatedImage
 {
     _animatedImage = animatedImage;
     _animatedImageRunLoopMode = NSRunLoopCommonModes;
+    
+    [self initializeAnimatedImage:animatedImage];
```

Comment:
Can you add a nil check here and make mark `initializeAnimatedImage:` as taking a non null animatedImage?